app/controllers/discovered_hosts_controller.rb

```diff
@@ -26,7 +26,7 @@ def index
       :organization,
       :model,
       :discovery_attribute_set
-    ], {:interfaces => :subnet}).paginate(:page => params[:page])
+    ], {:interfaces => :subnet}).paginate(:page => params[:page], :per_page => params[:per_page])
```

Comment:
please use `@hosts = resource_base_search_and_page.includes(....` so if something changes in core we won't have to update this as well.